client/src/components/ElementEditor/Header.js

```diff
@@ -198,12 +239,13 @@ Header.propTypes = {
   id: PropTypes.string,
   title: PropTypes.string,
   version: PropTypes.number,
-  isPublished: PropTypes.bool,
+  isLiveVersion: PropTypes.bool,
   elementType: PropTypes.string,
   fontIcon: PropTypes.string,
   actions: PropTypes.shape({
     handleDeleteBlock: PropTypes.func.isRequired,
-    handlePublishBlock: PropTypes.func
+    handlePublishBlock: PropTypes.func,
+    handleUnpublishBlock: PropTypes.func
```

Comment:
If you add a trailing comma then the next PR won't show this line as changed (like this one has)